lib/tasks/analyse_javascript.rake

```diff
@@ -0,0 +1,5 @@
+desc 'Analyse Javascript with JSHint and Javascript Code Style checker'
+task :analyse_javascript do
+  puts 'Running Javascript code analysis...'
+  system('npm run js') || raise('Javascript analysis failed')
```

Comment:
Do we need to raise here when we could rely on the exit code of the `npm` sub-process?